seungjong/controllers/likeController.js

```diff
@@ -0,0 +1,27 @@
+const likeService = require('../services/likeService');
+
+const like = async (req, res) => {
+    try {
+        const {userId, postId} = req.params;
+        console.log(`111111111:::userId::${userId},postId:::${postId}`);
+        
+        if( !userId || !postId) {
+            return res.status(400).json({message: "NOT_FOUND" });
+        }
+        const result = await likeService.like(userId, postId);
+        console.log(result.insertId);
+        if (result.insertId === 0) {
+            return res.status(200).json({message: "deleteLiked!"});
+        } else {
+            return res.status(200).json({message: "likeCreated!"});            
+        }
```

Comment:
```suggestion
        if (result.insertId === 0) {
            return res.status(200).json({message: "deleteLiked!"});
        } 
        
        return res.status(200).json({message: "likeCreated!"});            
```
else없이 작성되어도 똑같이 동작합니다. 또한 실제로는 else를 쓰면서 개발하는 경우는 많지 않습니다. 오히려 가독성을 해치고 대부분 else가 없어도 되는 로직이기 때문입니다.